crates/forge-runner/src/printing.rs

```diff
@@ -37,7 +43,60 @@ pub(crate) fn print_test_result(any_test_result: &AnyTestCaseSummary) {
         }
         _ => String::new(),
     };
-    println!("{result_header} {result_name}{fuzzer_report}{gas_usage}{result_msg}");
+
+    let used_resources = match any_test_result {
+        AnyTestCaseSummary::Single(TestCaseSummary::Passed { used_resources, .. }) => {
```

Comment:
Please add an issue to add support for fuzzing as well

Reply:
#1628